docs/src/misc.rst

```diff
@@ -322,6 +322,15 @@ API
         On Windows not all fields are set, the unsupported fields are filled with zeroes.
         See :c:type:`uv_rusage_t` for more details.
 
+.. c:function:: int uv_getrusage_thread(uv_rusage_t* rusage)
+
+    Gets the resource usage measures for the calling thread.
+
+    .. note::
+        Not supported on all platforms. May return `UV_ENOTSUP`.
+        On Windows not all fields are set, the unsupported fields are filled with zeroes.
+        See :c:type:`uv_rusage_t` for more details.
+
```

Comment:
The `.. versionadded::` label should be added

Reply:
Added. Hopefully I put the correct value but not sure.

Reply:
It should be `1.42.0` as this is a semver minor change.